tools/cli/admin.go

```diff
@@ -170,7 +170,45 @@ func newAdminShardManagementCommands() []cli.Command {
 				AdminDescribeTask(c)
 			},
 		},
-
+		{
+			Name:    "list-tasks",
```

Comment:
Not blocking, but lets add predicate for task ID at some point, based off discussion we had Friday